envs/envs_test.go

```diff
@@ -70,6 +70,27 @@ func (f fakeEnv) Local() bool {
 	return true
 }
 
+func (s *ScenvSuite) TestGenericContainerExposesHostAndPort(c *C) {
```

Comment:
This test passes even without your change. A test should be added in `docker_test.go` instead.

Reply:
I've deleted this test (to be honest, i've not ran the test because I haven't had a go development environment ready).

For a test in `docker_test.go` I would propose to test the docker integration more sophisticated (like with testcontainers, so we can test the interaction with real containers). I'm willing to send a PR, but it would take a day or two.

Reply:
Ok, let's commit as is for now.